src/test/java/org/folio/logintest/RefreshTest.java

```diff
@@ -0,0 +1,198 @@
+    PostgresClient.getInstance(vertx);
+
+    spec = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.URL, "http://localhost:" + mockPort)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", "accessToken=123; refreshToken=321")
+        .build();
+
+    specExpiredToken = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.URL, "http://localhost:" + mockPort)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", "accessToken=expiredtoken; refreshToken=321")
+        .build();
+
+    specBadRequestMissingAccessTokenCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", "refreshToken=321")
+        .build();
+
+    specBadRequestEmptyCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", "")
+        .build();
+
+    specBadRequestNoCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .build();
+
+    specDuplicateKeyCookie = new RequestSpecBuilder()
+        .setBaseUri("http://localhost:" + port)
+        .addHeader(XOkapiHeaders.TENANT, TENANT_DIKU)
+        .addHeader("Cookie", "accessToken=098;refreshToken=321;refreshToken=456")
+        .build();
+
+    TenantAttributes ta = new TenantAttributes().withModuleTo("mod-login-1.1.0");
+    vertx.deployVerticle(Mocks.class.getName(), mockOptions)
+        .compose(res -> vertx.deployVerticle(RestVerticle.class.getName(), options))
+        .compose(res -> TestUtil.postSync(ta, TENANT_DIKU, port, vertx))
+        .onComplete(context.asyncAssertSuccess());
+  }
+
+  @Test
+  public void testRefreshCreated(final TestContext context) {
+    RestAssured.given()
+        .spec(spec)
+        .when()
+        .post(REFRESH_PATH)
+        .then()
+        .log().all()
+        .statusCode(201)
+        .contentType("application/json")
+        .cookie(LoginAPI.REFRESH_TOKEN, RestAssuredMatchers.detailedCookie()
+            .value(Mocks.REFRESH_TOKEN)
+            // Account for time drift because we're using Now.Instant to compute max-age.
+            .maxAge(
+                allOf(greaterThan(Mocks.REFRESH_TOKEN_EXPIRATION - 2), lessThan(Mocks.REFRESH_TOKEN_EXPIRATION + 1)))
+            .path("/authn")
+            .httpOnly(true)
+            .secured(true))
+        .cookie(LoginAPI.ACCESS_TOKEN, RestAssuredMatchers.detailedCookie()
+            .value(Mocks.ACCESS_TOKEN)
+            // Account for time drift because we're using Now.Instant to compute max-age.
+            .maxAge(allOf(greaterThan(Mocks.ACCESS_TOKEN_EXPIRATION - 2), lessThan(Mocks.REFRESH_TOKEN_EXPIRATION + 1)))
+            .httpOnly(true)
```

Comment:
Please add

    .domain((String) null)  // not setting Domain disables subdomains
    .path((String) null)    // not setting Path allows all paths